modules/test/playwright/tests/search-experiences-web/main/sxpBlueprint.spec.ts

```diff
@@ -647,3 +651,167 @@ test.describe('Blueprint can be registered as a collection provider', () => {
 		});
 	});
 });
+
+test.describe('View links to Liferay Documentation', () => {
```

Comment:
To help readability of the playwright test reports, I'd name `test.describe` like a title rather than action. It'd help read more like `"category" > "test action"` instead `"action" > "action"`.

I'd name this something like `Documentation Links`.

Right now the tests read:
```
View links to Liferay Documentation › Blueprint and element editor have working links
```

After: 
```
Documentation Links › Blueprint and element editor have working links
```

I could see it reading well the other way around too like `"Test documentation links > Blueprint and element editor"` so I think overall mainly just keeping the `test.describe('')` and `test('')` blocks to a single action helps a lot so it's not like you're reading the title twice.